tests/components/do_verify_email.test.jsx

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+import {shallow, configure} from 'enzyme';
+
+import Adapter from 'enzyme-adapter-react-16';
+
+import {verifyUserEmail as VerifyUserMail} from 'mattermost-redux/actions/users';
+
+import DoVerifyEmail from 'components/do_verify_email/do_verify_email.jsx';
+
+configure({adapter: new Adapter()});
+
+describe('components/DoVerifyEmail', () => {
+    const requiredProps = {
+        location: {
+            query: '',
+            token: ''
```

Comment:
Is this correct? It looks like `token` should be nested in `query` per this access above `this.props.location.query.token`